avatars/vrarmik/LegsManager.js

```diff
@@ -146,6 +149,10 @@ class Leg {
         .multiply(downHalfRotation)
         .premultiply(Helpers.getWorldQuaternion(this.lowerLeg, localQuaternion2).invert());
       Helpers.updateMatrixMatrixWorld(this.foot);
+
+
+      this.toe.quaternion.copy(footRotation).multiply(downHalfRotation).premultiply(Helpers.getWorldQuaternion(this.lowerLeg, localQuaternion2).invert());
```

Comment:
I'm not sure if this is correct; we should test it by hacking IK to be enabled: https://github.com/webaverse/app/blob/5af9f875958852454c83bee49f18cfcbb044ac49/avatars/avatars.js#L2156